docs/security/host-certs.md

```diff
@@ -108,6 +115,20 @@ generated above.
         root@host # cp %RED%<PATH TO KEY>%ENDCOLOR% /etc/grid-security/hostkey.pem
         root@host # chmod 400 /etc/grid-security/hostkey.pem
 
+### Request certificates in bulk using osg-incommon-cert-request
+
+1. Find your institution-specific InCommon contact
+   (e.g. [UW-Madison InCommon contact](https://it.wisc.edu/about/office-of-the-cio/cybersecurity/security-tools-software/server-certificates/)),  
+1. Ask for a Department Registration Authority user with SSL auto approve enabled. 
+1. Ask for an email invitation for a Client Certificate. Download your InCommon user certificate and split the .p12 file into your private key and your certificate.
+1. Request a certificate and generate the associated private key using the osg-incommon-cert-request:
+
+        :::console
+        user@host $ osg-incommoncert-request --username <INCOMMONLOGIN> \
+                    --cert <USERCERT> \
+                    --pkey <USERPRIVKEY> \ 
+                    --hostname <HOSTNAME>
+
```

Comment:
Also include an example alternative name, and show how to use the bulk filename input.

There's going to have to be another option or options for setting the organization & department codes, but those could be added to the documentation later.